### PR TITLE
[New badge] AppleStore version

### PR DESCRIPTION
**Pull request by javierugarte**
captura de pantalla 2016-12-15 a las 21 09 18

New badge with Apple Store version.

In my case, I want use to compare Github release with AppleStore last version.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1685,6 +1685,33 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Apple Store version
+camp.route(/^\/applestore\/v\/(.+)\.(svg|png|gif|jpg|json)$/,
```
Comment (paulmelnikow) [inline server.js]: Could I ask you to change this to `itunes`?

File: server.js [hunk 2]
```diff
@@ -1685,6 +1685,33 @@ cache(function(data, match, sendBadge, request) {
...
+cache(function(data, match, sendBadge, request) {
+  var bundleId = match[1];  // eg, `324684580`
+  var format = match[2];
+  var apiUrl = 'https://itunes.apple.com/lookup?id=' + bundleId;
+  var badgeData = getBadgeData('AppleStore', data);
```
Comment (paulmelnikow) [inline server.js]: Could I ask you to change this to `App Store` (with space)?

**Comment by paulmelnikow** [comment]
Thanks so much for this! Would love to get it merged. I asked for a couple minor changes.

File: server.js (hunk 2 above)
Comment (paulmelnikow) [inline server.js]: On second thought, it seems `itunes app store` would be more consistent with the existing badge formats (c.f. `chrome web store`).

File: server.js (hunk 1 above)
Comment (javierugarte) [inline server.js]: Sure, itunes is the online store

File: server.js (hunk 2 above)
Comment (javierugarte) [inline server.js]: ok

**Comment by javierugarte** [comment]
Done

**Comment by paulmelnikow** [comment]
Thanks so much!

**Pull request merged**